user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper/StepperTutorial.md

```diff
@@ -0,0 +1,91 @@
+---
+uid: TutorialStepper
+---
+
+# Using the stepper component in a low-code app
+
+In this tutorial, you will learn how to add and configure a stepper component in a low-code app, by means of an example where the stepper component is used to help with managing incidents and their different states.
+
+The content and screenshots for this tutorial have been created in DataMiner version 10.3.10.
+
+## Overview
+
+- [Step 1: Set up the data](#step-1-set-up-the-data)
+- [Step 2: Create the components](#step-2-create-the-components)
+- [Step 3: Add the data](#step-3-add-the-data)
+- [Step 4: Add a save action](#step-4-add-a-save-action)
+- [Step 5: Style the stepper](#step-5-style-the-stepper)
+- [Step 6: Use the low-code app](#step-6-use-the-low-code-app)
+
+## Step 1: Set up the data
+
+Before you can begin to create the app, you will need to add some data to your system that you can visualize in the app. To do so, deploy the [Incident Manager package](https://catalog.dataminer.services/catalog/4383) from the catalog.
+
+The package will create an "Incident" [DOM definition](xref:DomDefinition) and some [DOM instances](xref:DomInstance) using it.
+
+> [!TIP]
+> Deploying a package is very similar to deploying a DataMiner connector. See [Deploying a DataMiner connector to your system](xref:Deploying_A_DataMiner_Connector_to_your_system).
+
+## Step 2: Create the components
+
+Now that you have deployed the package with the data, you can create the app to visualize this data.
+
+1. [Create a new application](xref:Creating_custom_apps).
+
+1. Click the pencil icon next to the initial page of the application in order to edit it.
+
+1. Hover the mouse pointer over the bar with the "+" on the left until it expands to show the Visualizations pane, and add a [stepper](xref:DashboardStepper), a [form](xref:DashboardForm), and a [table](xref:DashboardTable) component by dragging them onto the page.
+
+   You can find the stepper and form components in the *General* category and the table in the *Tables* category.
+
+The end result of this step should look like this:
+
+![Components](~/user-guide/images/StepperComponents.png)
```

Comment:
This screenshot and the one for the next step were taken from a fairly large window, but as the image gets resized to fit in the docs layout, the text on docs will be very small and hard to read. Would it be possible to replace these with screenshots of smaller windows, so that the user will be able to see everything more clearly?